x-pack/auditbeat/module/system/socket/guess/creds.go

```diff
@@ -19,16 +19,12 @@ import (
 	"github.com/elastic/elastic-agent-libs/mapstr"
 )
 
-/*
-struct mq_attr {
-	long mq_flags;
-	long mq_maxmsg;
-	long mq_msgsize;
-	long mq_curmsgs;
-	long __reserved[4];
-};
-*/
-import "C"
+type MqAttr struct {
+	_       int
+	Maxmsg  int
+	Msgsize int
+	_       int
+}
 
```

Comment:
This is changing the size of the structure, I'd keep the reserved bits in, as mq_open expects attr to be 8 longs